src/Views/OperatingSystemView.vala

```diff
@@ -127,13 +152,13 @@ public class About.OperatingSystemView : Gtk.Grid {
         button_grid.set_child_secondary (settings_restore_button, true);
 
         var software_grid = new Gtk.Grid () {
-            column_spacing = 12,
+            column_spacing = 6,
```

Comment:
I was actually wondering if we should go in the opposite direction; as-is, this large graphic is all up on the text; I was playing with 24 locally and that felt like it let the logo breathe a little better.

```suggestion
            column_spacing = 24,
```

![Screenshot from 2021-01-04 19-05-12](https://user-images.githubusercontent.com/611168/103598589-38e2b100-4ec0-11eb-88c9-168c4f6bed29.png)